x-pack/platform/plugins/shared/streams_app/public/components/data_management/stream_detail_enrichment/processors_list.tsx

```diff
@@ -6,26 +6,47 @@
  */
 
 import React from 'react';
-import { EuiDraggable } from '@elastic/eui';
-import { EditProcessorPanel, type EditProcessorPanelProps } from './processors';
+import { EuiDraggable, EuiTimelineItem } from '@elastic/eui';
+import { css } from '@emotion/react';
+import { ProcessorConfiguration, ProcessorConfigurationProps } from './processors';
+import { ProcessorStatusIndicator } from './processor_status_indicator';
 
 export const DraggableProcessorListItem = ({
   idx,
-  disableDrag,
+  isDragDisabled,
   ...props
-}: Omit<EditProcessorPanelProps, 'dragHandleProps'> & { idx: number; disableDrag: boolean }) => (
-  <EuiDraggable
-    index={idx}
-    spacing="m"
-    draggableId={props.processorRef.id}
-    hasInteractiveChildren
-    customDragHandle
-    isDragDisabled={disableDrag}
-    css={{
-      paddingLeft: 0,
-      paddingRight: 0,
-    }}
-  >
-    {(provided) => <EditProcessorPanel {...props} dragHandleProps={provided.dragHandleProps} />}
-  </EuiDraggable>
-);
+}: Omit<ProcessorConfigurationProps, 'dragHandleProps'> & {
+  idx: number;
+  isDragDisabled: boolean;
+}) => {
+  return (
+    <EuiTimelineItem
+      verticalAlign="top"
+      icon={<ProcessorStatusIndicator processorRef={props.processorRef} />}
+      css={css`
+        [class*='euiTimelineItemEvent'] {
+          min-width: 0;
+        }
+        [class*='euiTimelineItemIcon-top'] {
+          translate: 0 9px; // (50px - 32px) / 2 => Height of the block minus the avatar size to center the item
+        }
+      `}
```

Comment:
Required CSS ad-hoc customization to center and align the timeline steps as required. Not fancy, but the default component styling doesn't support direct customization.